packages/compass-schema-validation/src/stores/store.ts

```diff
@@ -0,0 +1,117 @@
+import type { Store } from 'redux';
+import { createStore, applyMiddleware } from 'redux';
+import type { ThunkDispatch } from 'redux-thunk';
+import thunk from 'redux-thunk';
+import type { RootAction, RootState } from '../modules';
+import reducer from '../modules';
+import toNS from 'mongodb-ns';
+import { namespaceChanged } from '../modules/namespace';
+import { dataServiceConnected } from '../modules/data-service';
+import { fieldsChanged } from '../modules/fields';
+import { serverVersionChanged } from '../modules/server-version';
+import { activateValidation } from '../modules/validation';
+import { editModeChanged } from '../modules/edit-mode';
+import {
+  localAppRegistryActivated,
+  globalAppRegistryActivated,
+} from '@mongodb-js/mongodb-redux-common/app-registry';
+import semver from 'semver';
+import type { CollectionTabPluginMetadata } from '@mongodb-js/compass-collection';
+import type { AppRegistry } from 'hadron-app-registry';
+import type { DataService } from 'mongodb-data-service';
+import type { MongoDBInstance } from '@mongodb-js/compass-app-stores/provider';
+
+/**
+ * The lowest supported version.
+ */
+const MIN_VERSION = '3.2.0';
+
+// Exposed for testing
+export function createBareStore(): Store<RootState, RootAction> {
```

Comment:
Exporting it for testing is fine, but we do call those methods just `configureStore` usually

Reply:
Ahhh, gotcha, sorry, I think I just didn't notice that the `configureStore` was still there